test/distributed/org/apache/cassandra/distributed/test/accord/AccordCQLTest.java

```diff
@@ -31,10 +31,16 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
+import org.apache.cassandra.cql3.CQLTester;
 
 import accord.primitives.Unseekables;
 import accord.topology.Topologies;
 import org.apache.cassandra.cql3.functions.types.utils.Bytes;
+import org.apache.cassandra.db.marshal.*;
```

Comment:
nit: unused?

Reply:
I am not even sure how that can happen with my settings. Optimized the imports.